topcoffea/modules/CorrectedJetsFactory.py

```diff
@@ -333,330 +336,328 @@ def build(self, jets, lazy_cache, isdata = False):
         out_dict[self.name_map["JetPt"] + "_jec"] = out_dict[self.name_map["JetPt"]]
         out_dict[self.name_map["JetMass"] + "_jec"] = out_dict[self.name_map["JetMass"]]
 
-        if not isdata: # Data does not need JER or uncertainties
```

Comment:
So we don't need a check for data here?

Reply:
If it is data, `ApplyJetCorrections` from topeft won't require JER and JEC uncertainties to be computed. This is to expose to the user this choice, and don't hardcode it in an auxiliary class
https://github.com/TopEFT/topeft/blob/clib_run3/topeft/modules/corrections.py#L1427
https://github.com/TopEFT/topeft/blob/clib_run3/topeft/modules/corrections.py#L262-L285